examples/ssx-test-react-dapp/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ssx-test-react-dapp",
+  "private": true,
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@rainbow-me/rainbowkit": "^0.7.2",
+    "@spruceid/ssx-react": "*",
```

Comment:
Update the @spruceid/ssx-react dependency to point to /dist and avoid the local installation error

Reply:
Done